app/templates/curriculum-inventory-reports.hbs

```diff
@@ -81,7 +81,7 @@
     <div class="list">
       {{#if (is-fulfilled this.reports)}}
         <CurriculumInventoryReportList
-          @program={{await this.selectedProgram}}
+          @reports={{get (await this.selectedProgram) "curriculumInventoryReports"}}
```

Comment:
I'm not sure this away is in the right place. Shouldn't it be `@reports={{await(get this.selectedProgram "curriculumInventoryReports")}}`?

Reply:
that doesn't work. to be on the safe side, i added another `await` for the linked reports.

```
@reports={{await (get (await this.selectedProgram) "curriculumInventoryReports")}}
```